owslib/util.py

```diff
@@ -591,6 +591,10 @@ def dump(obj, prefix=''):
 def getTypedValue(data_type, value):
     '''Utility function to cast a string value to the appropriate XSD type. '''
 
+    # If the default value is empty
+    if value is None:
+        return value
```

Comment:
```suggestion
        return
```

Reply:
OK for me. Thanks.

Reply:
The build failed, but the error is not connected to this pull request.